x-pack/plugin/core/src/main/resources/logs-template.json

```diff
@@ -0,0 +1,16 @@
+{
+  "index_patterns": ["logs-*-*"],
+  "priority": 100,
```

Comment:
@dakrone Why is this set to 100? Shouldn't we set it to 1? @jonathan-buttner FYI

Reply:
@ruflin hah, you already asked that in https://github.com/elastic/elasticsearch/pull/57629#discussion_r435415763 :) We need space in order for templates to be allowed at a lower priority than these built-in ones (for example, someone installing a template for `*` at priority 10)

Reply:
🤦